apk-generator/v2/scripts/fdroid_build_apk.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+./gradlew assembleRelease --info
```

Comment:
`./gradlew assembleRelease` will assemble all release builds.
If you are making different script for fdroid and google play then you should use `./gradlew assembleFdroidrelease` and `./gradlew assemblegoogleplayrelease`